qualtran/bloqs/mod_arithmetic/mod_division.py

```diff
@@ -632,6 +632,9 @@ def build_composite_bloq(
         bb.free(f)
         return {'x': x, 'm': m}
 
+    def adjoint(self) -> 'KaliskiModInverse':
+        return evolve(self, uncompute=self.uncompute ^ True)
```

Comment:
I have a strong preference for writing this as `not self.uncompute`

Reply:
done